schema/core/primitive/acyclic_graph/base_node.json

```diff
@@ -0,0 +1,42 @@
+{
+    "schemaId": "core-primitive-graph-base-node",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "basic node schema (directed acyclic graph)",
+    "type": "object",
+    "properties": {
+        "flowchartId": {
+            "description": "Unique flowchart ID of node",
+            "type": "string"
+        },
+        "next": {
+            "description": "Flowchart ID of next node",
+            "type": "string"
+        },
+        "head": {
+            "description": "Whether node is head node or not",
+            "type": "boolean"
+        }
+    },
+    "required": [
+        "flowchartId"
+    ],
+    "definitions": {
+        "string-or-array-of-strings": {
+            "oneOf": [
+                {
+                    "type": "string"
+                },
+                {
+                    "$ref": "../array_of_strings.json"
+                }
+            ]
+        },
+        "flowchart-id": {
```

Comment:
1. Should we define the above property on the line 7 through this?
2. Having a "-" in the name may make the schemaIds resolution a bit clunky unless we change the separator in the ID